thoth/storages/graph/postgres.py

```diff
@@ -4132,6 +4132,23 @@ def delete_kebechet_github_app_installations(
             query.delete()
             return count
 
+    def get_kebechet_github_installations_software_stack_count_all(
+        self,
+    ):
+        """Get number of Kebechet maintained software stacks."""
+        with self._session_scope() as session:
+            query = (
+                session.query(KebechetGithubAppInstallations)
+                .filter(KebechetGithubAppInstallations.is_active.is_(True))
```

Comment:
Could the query be parametrized with `is_active: Optional[bool] = None`? It would fit with other queries and we can eventually obtain possible active/inactive installations count.

Reply:
Sure great idea! Thanks @fridex!